### DISCUSSION TITLE
Cannot scroll in pinned menu

### DISCUSSION THREAD

**Discussion by DylanBT928** [community/community#106709] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

## Bug:

When I click "Customize your pins" to add/remove my repositories, I cannot scroll up or down. I am able to scroll on the actual pin menu, but I cannot scroll on the main page. 

This becomes an issue when I click the "Customize your pins" button before scrolling down. I cannot see half of my repositories and even the "Save pins" button.

## How to reproduce:

1. Visit your profile page
2. Scroll down until "Customize your pins" is on the lower third of your screen
3. Click the "Customize your pins" button
4. Try to scroll down

*The screen only jumps down to let you view the entire menu when the "Customize your pins" button is at the very bottom of your screen.*

**Comment by mmascheroni** [comment]
The same thing happened to me yesterday, I had to open the custome pins leaving it at the top and clicking there allows you to scroll and the save pins button appears
![1](https://github.com/community/community/assets/106355008/cfeff5b6-f085-46ef-8b36-2aa126ebb25b)

**Comment by Dnkfff** [comment]
So basically, when you click on `Customize your pins` button, you cannot scroll in your profile. I'm not sure it would be fixed, but if you really want Github to work on it, you could apply a ticket to `bug report`. I'm not sure you should scroll everywhere, while searching your repositories. 
But it worth a shot, you can submit your bug here https://bounty.github.com/
May luck be with you!

**Comment by HaigEmirzian** [comment]
Having the same problem, but I was able to minimize my page and the save button appeared. Regardless, this is a temporary fix and users should be able to scroll on the main page when trying to click save.
